docs/usage/configuration-options.md

```diff
@@ -1834,11 +1834,13 @@ For example, GitHub might automerge a Renovate branch even if it's behind the ba
 
 ## platformCommit
 
-Supports only GitHub App mode and not when using Personal Access Tokens.
+If you're authenticating Renovate as an app, then you can use this option to automatically sign the commits that Renovate creates.
+This option only works if you:
 
-To avoid errors, `gitAuthor` or `gitIgnoredAuthors` should be manually adjusted accordingly.
+- Use GitHub App mode, and
```

Comment:
What does "GitHub App" mode refer to? It's in the original text.

I suppose it means: "WhiteSource Hosted GitHub App" (The version of Renovate that you can install into your GitHub account)?